test/ProgressOnderwijsUtils.Tests/RandomHelperTest.cs

```diff
@@ -60,7 +60,9 @@ public void ImplicitlyInsecure()
         {
             var randomHelper1 = RandomHelper.ImplicitlyInsecure();
             var randomHelper2 = RandomHelper.ImplicitlyInsecure();
+            var (randomHelper3, randomHelper4) = (RandomHelper.ImplicitlyInsecure(), RandomHelper.ImplicitlyInsecure());
             PAssert.That(() => randomHelper1.GetInt32() != randomHelper2.GetInt32());
+            PAssert.That(() => randomHelper3.GetInt32() == randomHelper4.GetInt32());

```

Comment:
Je zou er ook twee losse testen van kunnen maken - typisch wil je dat een test maar 1 ding test, en dat de naam van de test weergeeft wat er getest wordt. Maar zo ook prima natuurlijk

Reply:
Done. De naamgeving wijkt nu wel wat af van de overige tests in de file, maar het is idd een stuk duidelijker zo.